tests/lax_numpy_test.py

```diff
@@ -2209,14 +2209,14 @@ def testIx_(self, rng_factory, shapes, dtypes):
              jtu.format_shape_dtype_string(a_shape, a_dtype),
              jtu.format_shape_dtype_string(q_shape, q_dtype),
              axis, keepdims, interpolation),
-         "a_rng": jtu.rand_default(), "q_rng": q_rng, "op": op,
+         "a_rng": jtu.rand_default, "q_rng": q_rng, "op": op,
          "a_shape": a_shape, "a_dtype": a_dtype,
          "q_shape": q_shape, "q_dtype": q_dtype, "axis": axis,
          "keepdims": keepdims,
          "interpolation": interpolation}
         for (op, q_rng) in (
-          ("percentile", jtu.rand_uniform(low=0., high=100.)),
-          ("quantile", jtu.rand_uniform(low=0., high=1.)),
+          ("percentile", lambda: jtu.rand_uniform(low=0., high=100.)),
```

Comment:
Consider spelling this as `partial(jtu.rand_uniform, low=0., high=100.)`. Up to you.

Reply:
I think the lambda's a little clearer, since we're fully applying the function inside the q_rng callable.